packages/design-tokens/docs/DocsComponents.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
What's the TLDR on why these are in a few files at the top?

Reply:
Was curious about that too, I'm assuming the pattern is if its at the top it applies to the whole component? @dougmacknz is my read on that right?

Reply:
That's right, this will disable the rule for the whole file. Also notice it's `eslint-disable` instead of `eslint-disable-next-line`.

Reply:
Ah my bad, I know they're disabling the whole file. I just meant why are there so many disabling the whole file?

Reply:
Oh I was replying to @mcwinter07. Hmm, on second look this one should just be on the line, but for the others it's really common to do invalid links in stories so I just added it to the top once instead of having to do it for each line.